bash_completion.sh

```diff
@@ -159,12 +159,15 @@ _buildtest ()
       ;;
 
     report|rt)
-      local opts="--end --failure --filter --format --help --helpfilter --helpformat --latest --no-header --oldest --pager --passed --start --terse  -e -f -h -n -p -s -t clear list summary"
-      COMPREPLY=( $( compgen -W "$opts" -- $cur ) )
-      case "${COMP_WORDS[2]}" in summary)
-        local opts="-d -h --detailed --help --pager"
-        COMPREPLY=( $( compgen -W "${opts}" -- $cur ) )
-        ;;
+      local opts="--end --failure --filter --format --help --helpfilter --helpformat --latest --no-header --oldest --pager --passed --start --terse  -e -f -h -n -p -s -t clear list summary sm"
+      COMPREPLY=( $( compgen -W "${opts}" -- $cur ) )
+      case "${COMP_WORDS[2]}" in summary|sm)
+        local opt="-d -h --detailed --help"
+        COMPREPLY=( $( compgen -W "${opt}" -- $cur ) );;
+      esac
+      case "${COMP_WORDS[3]}" in summary|sm)
+        local opt="-d -h --detailed --help"
+        COMPREPLY=( $( compgen -W "${opt}" -- $cur ) );;
       esac
```

Comment:
can you please remove this part or revert it back to original this is logic for `summary` i do think you just need to remove the `--pager` option

Reply:
i think the `case` statement that you have are coming from the other MR that you are working on with `--color`

Reply:
I moved it back and I think this is ready to merge. I am waiting on this one so that way I can get some of the code for Issue 1190 branch

Reply:
Can you remove the `sm`  from bash completion from this PR and add this in separate PR when you add the alias